sdk/keyvault/keyvault-certificates/package.json

```diff
@@ -95,7 +95,8 @@
   "devDependencies": {
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/identity": "^1.1.0-preview",
-    "@azure/keyvault-secrets": "^4.1.0-preview.1",
+    "@azure/keyvault-keys": "^4.1.0-preview.1",
```

Comment:
Didnt you remove the keys dependencies in another PR?
It is being added back here

Reply:
thank you!